js/views/startScreen.js

```diff
@@ -89,7 +89,7 @@ var StartScreen = React.createClass({
 
     var titleMetadata = (<div className={titleClass} style={titleStyle}>{this.props.contentTree.title}</div>);
     var iconName = (this.props.controller.state.playerState == CONSTANTS.STATE.END ? "replay" : "play");
-    var descriptionMetadata = (<div className={descriptionClass} ref="description" style={descriptionStyle}>{this.state.descriptionText}</div>);
+    var descriptionMetadata = (<div className={descriptionClass} ref="description" style={descriptionStyle}>{this.props.contentTree.description}</div>);
```

Comment:
@brianfwong we also need to leave this has a state variable. This update would break the description being truncated to 2 lines when the width of the player is resized. Use `componentWillReceiveProps()` to update `this.state.descriptionText` when `this.props.contentTree.description` is updated.